### DISCUSSION TITLE
Add a new standard?

### DISCUSSION THREAD

**Discussion by barrykaplan** [OWASP/OpenCRE#484] [Q&A]
What would be the process for adding a mapping to a new standard? I see some mappings in the yml files, but also in PRs specific standards a referred to in code. 

Did I miss something in docs/ ?

**Comment by northdpole** [comment]
Hey @barrykaplan , thanks for openinga ticket, you usually either:

1. submit a PR with either yaml files
2. Open a ticket with a Csv containing mappings
3. Add cre links to your standard and ask is to parse it

But for the next version we are refactoring this to make it easier. 
Would you have time for a feedback session to help brainstorm?

**Reply by barrykaplan** [reply]
Thanks @northdpole. I was expecting to need to a bit more work. Sure, I’m happy to discuss. 

Another area I was thinking about was “private” mappings to an organizations internal controls. That is why I was asking about the process of (I guess) what happens after the cvs mapping exists.
